docker/jenkins/Dockerfile.centos8-x86_64

```diff
@@ -1,24 +1,27 @@
-FROM fedora:28
+FROM centos:8
+
+RUN set -x \
+      && yum install epel-release -y \
+      && yum install dnf-plugins-core -y \ 
+      && yum config-manager --set-enabled PowerTools \
+      && yum update -y
 
 RUN yum install -y \
     ant \
     boost-devel \
     bzip2-devel \
     clang-devel \
-    curl \
     fakeroot \
     fuse-libs \
     gcc \
     gcc-c++ \
     git \
-    gpg \
```

Comment:
We need this to sign builds. I think the corresponding CentOS 8 package is gnupg1.

Reply:
This is already pulled in implicitly as another package's dependency, but doesn't hurt to explicitly add it! I have done so, with `gnupg2`.